lib/actions/adapterActions.js

```diff
@@ -578,14 +578,15 @@ function handleDeviceEvent(device, getState, handleFn) {
 
 function validatePort(dispatch, getState, port) {
     const adapterToUse = getState().app.adapter.api.adapters.find(
-        x => (x.state.port === port.comName),
+        x => (x.state.port === port.comName ||
+            x.state.port === port.comName.replace('tty', 'cu')),
```

Comment:
Perhaps add a brief comment on why this is needed?

Reply:
Sure, will add.